src/components/common/ColonyMembers/MembersFilter/MembersFilter.tsx

```diff
@@ -66,9 +66,11 @@ const MembersFilter = ({ handleFiltersCallback, isRoot }: Props) => {
                 </span>
                 {showReset && (
                   <Button
+                    disabled={!isValid || isSubmitting}
                     text={MSG.reset}
+                    type="submit"
```

Comment:
Don't think `type='submit'` is needed either given there's no submit handler on the form.